Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM mhart/alpine-node:4
+# docker-kassy
+# -----------------------------------------------------------------------------
+# Alpine Linux containerised version of Kassy ready for deployment
+# -----------------------------------------------------------------------------
+# Make sure to map your config.json file into /kassy!
+# see: kassy/doc/integrations/Facebook.md for the file format
+MAINTAINER Matt Hartstonge <matt@mykro.co.nz>
+
+COPY ./ /kassy
+RUN apk --no-cache add \
+        bash \
+        g++ \
+        make \
+        python \
+    && mv /kassy/kassy /usr/bin/kassy \
+    && chmod 744 /usr/bin/kassy \
+    && cd /kassy \
+    && npm install \
+    && apk del \
+        python \
+        make \
+        g++
+
+WORKDIR ["/kassy"]
+VOLUME ["/kassy/modules/"]
+ENTRYPOINT ["kassy"]
+CMD ["--fb"]
```

Comment:
That's correct. Solely due to documentation pointing to FB config first.
But again, this can be changed to --test which might make a bit more sense?
If anything is added to the end of the docker run command, it will override
whatever is in the dockerfile CMD setting.

On Sat, 14 May 2016 at 10:40 PM, Matthew Knox <notifications@github.com>
wrote:

> In Dockerfile
> <https://github.com/concierge/Kassy/pull/157#discussion_r63274234>:
>
> > +        g++ \
> > +        make \
> > +        python \
> > +    && mv /kassy/kassy /usr/bin/kassy \
> > +    && chmod 744 /usr/bin/kassy \
> > +    && cd /kassy \
> > +    && npm install \
> > +    && apk del \
> > +        python \
> > +        make \
> > +        g++
> > +
> > +WORKDIR ["/kassy"]
> > +VOLUME ["/kassy/modules/"]
> > +ENTRYPOINT ["kassy"]
> > +CMD ["--fb"]
>
> So facebook is default?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/concierge/Kassy/pull/157/files/08df1c7c3da0c19c769677ac5e6dd8ef2f5daabb#r63274234>
>